README.adoc

```diff
@@ -172,10 +276,12 @@ If set to `true` and given a collection is not set in the map, an empty collecti
 Special field which can be used to determinate equality of 2 instances. When it is not defined the String representation of the 2 objects is used for `equals`
 - enumMappingMethod (default `name`)
 When an enum type is defined as a field, we need to define what type of representation is coded in the map.
+- mapNullToOptionalAbsent (default `false`)
+When it is true, when the value not set or absent in the given map it will return Optional.isPresent() false.
 
 == Supported options in MapBuilderProxy
 
-The MapBuilderProxy's builder supports additional options. For example:
+The MapBuilderProxy's builder supports all options from MapProxy and additional options. For example:
```

Comment:
"The MapBuilderProxy's builder supports all of the options of MapProxy with a couple of additions" ?